tests/test_event_parser.py

```diff
@@ -40,7 +47,12 @@ def parse(channel, version):
     return parser.parse(["tests/resources/test_events.yaml"], version, channel)
 
 
-def test_channel_version_ignore():
+@patch("os._exit")
+@patch(
+    "probe_scraper.parsers.third_party.shared_telemetry_utils.ParserError.eventual_errors"
```

Comment:
`tests/resources/test_events.yaml` intentionally contains eventual errors, this prevents those from raising an exception when pytest exits.